packages/core/test/model/chain/oasis.testnet.test.ts

```diff
@@ -0,0 +1,33 @@
+import { expect } from 'chai'
+import { TEST_ADDRESS, TEST_TX } from './defaults'
+import { OasisEmeraldTestnet } from '../../../src'
+
+describe('Oasis Emerald Chain', () => {
+  it('getChainId', () => {
+    expect(OasisEmeraldTestnet.chainId).to.equal(42261)
+  })
```

Comment:
Please put tests in the oasis.test.ts folder, because we test everything in that way.

Reply:
Put it in the same folder in the 'oasis.test.ts'.